cluster/common.sh

```diff
@@ -80,7 +80,9 @@ function create-kubeconfig() {
   fi
 
   "${kubectl}" config set-cluster "${CONTEXT}" "${cluster_args[@]}"
-  "${kubectl}" config set-credentials "${CONTEXT}" "${user_args[@]}"
+  if [ -n "${user_args[@]:-}" ]; then
```

Comment:
in my limited testing, I think using `[` instead of `[[` is what broke this for GCE, though I can't entirely explain why.